plugins/statistics.koplugin/main.lua

```diff
@@ -1872,7 +1872,7 @@ function ReaderStatistics:getDatesFromAll(sdays, ptype, book_mode)
             local stop_month = os.time{year=year_end, month=month_end, day=1, hour=0, min=0 }
             table.insert(results, {
                 date_text,
-                T(N_("%1 (%2 page)", "%1 (%2 pages)", tonumber(result_book[2][i])), datetime.secondsToClockDuration(user_duration_format, tonumber(result_book[3][i]), false, true), tonumber(result_book[2][i])),
+                T(N_("%1 (%2 page)", "%1 (%2 pages)", tonumber(result_book[2][i])), datetime.secondsToClockDuration(user_duration_format, tonumber(result_book[3][i]), false), tonumber(result_book[2][i])),
```

Comment:
What exactly is this saying? Is the first meant to be `%1 (1 page)`?

Reply:
I think a mistake from when the workings of `gettext` were a mystery, a mystery compounded by the order of the variables, left over from the statistics PR. It should say this, I think:

```lua
T(N_("%2 (1 page)", "%2 (%1 pages)", tonumber(result_book[2][i])), datetime.secondsToClockDuration(user_duration_format, tonumber(result_book[3][i]), false), tonumber(result_book[2][i])),
```

The mismatch between `tonumber(result_book[2][I])` in the N_() and `tonumber(result_book[3][I])` was there before even that, still not sure why they were different but left it alone

Reply:
Would you mind fixing it in a mini-PR if your mind's still in tune with this code?

Reply:
No problem, but as a sanity check, is the corrected line I posted actually correct?

Reply:
Not entirely, always write it sensibly, i.e., it should look like `"%1 (1 page)", "%1 (%2 pages)"`.

It's the code that should adapt, not what's given to the translators. ^_^

Reply:
I always always forget that the N_() does nothing except choose one of the strings depending on the number given in the second arg